wd.c

```diff
@@ -366,6 +366,14 @@ void wd_drv_unmap_qfr(handle_t h_ctx, enum uacce_qfrt qfrt)
 		munmap(ctx->qfrs_base[qfrt], ctx->qfrs_offs[qfrt]);
 }
 
+unsigned long wd_ctx_get_region_size(handle_t h_ctx, enum uacce_qfrt qfrt)
+{
+	struct wd_ctx_h *ctx = (struct wd_ctx_h *)h_ctx;
+	if (!ctx || qfrt >= UACCE_QFRT_MAX)
+			return 0;
+	return ctx->qfrs_offs[qfrt];
+}
+
```

Comment:
A new API is added in libwd. How do you use it? Could it be used in your test case?

Reply:
Qm user driver needs region size as device resetting,  but driver layer can not find ctx, so add a new API to get region size based h_ctx.